common/src/main/java/xyz/wagyourtail/jsmacros/client/access/IChunkSection.java

```diff
@@ -0,0 +1,9 @@
+package xyz.wagyourtail.jsmacros.client.access;
+
+public interface IChunkSection {
+
+    short getNonEmptyBlockCount();
```

Comment:
accessor methods should start with `jsmacros_` so that other mods creating the same method don't break shit